src/Aspire.Hosting.Qdrant/QdrantBuilderExtensions.cs

```diff
@@ -27,20 +27,22 @@ public static class QdrantBuilderExtensions
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
     /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency</param>
     /// <param name="apiKey">The parameter used to provide the API Key for the Qdrant resource. If <see langword="null"/> a random key will be generated as {name}-Key.</param>
-    /// <param name="port">The host port of Qdrant database.</param>
+    /// <param name="grpcPort">The host port of gRPC endpoint of Qdrant database.</param>
+    /// <param name="httpPort">The host port of REST endpoint of Qdrant database.</param>
     /// <returns>A reference to the <see cref="IResourceBuilder{QdrantServerResource}"/>.</returns>
     public static IResourceBuilder<QdrantServerResource> AddQdrant(this IDistributedApplicationBuilder builder,
         string name,
         IResourceBuilder<ParameterResource>? apiKey = null,
-        int? port = null)
+        int? grpcPort = null,
+        int? httpPort = null)
     {
         var apiKeyParameter = apiKey?.Resource ??
             ParameterResourceBuilderExtensions.CreateDefaultPasswordParameter(builder, $"{name}-Key", special: false);
         var qdrant = new QdrantServerResource(name, apiKeyParameter);
         return builder.AddResource(qdrant)
             .WithImage(QdrantContainerImageTags.Image, QdrantContainerImageTags.Tag)
-            .WithHttpEndpoint(port: port, targetPort: QdrantPortGrpc, name: QdrantServerResource.PrimaryEndpointName)
-            .WithHttpEndpoint(port: port, targetPort: QdrantPortHttp, name: QdrantServerResource.RestEndpointName)
+            .WithHttpEndpoint(port: grpcPort, targetPort: QdrantPortGrpc, name: QdrantServerResource.PrimaryEndpointName)
+            .WithHttpEndpoint(port: httpPort, targetPort: QdrantPortHttp, name: QdrantServerResource.RestEndpointName)
```

Comment:
Is it confusing that the `httpPort` goes to the `rest` endpoint, and not the endpoint that is named `http`?

Maybe we should either rename the endpoints, or rename these port parameters?

Reply:
I thought about this as developing. I was trying to match the way the Qdrant docs refer to the exposed API endpoints (as gRPC and REST).  But the more I think about it, it's mixing weird names.  

As you re-raise the question here @eerhardt , I think maybe representing the endpoints what they are (grpc and http) eliminating the 'high level' speak is maybe better...naming them clearly for what the endpoint communicates.

Thoughts? Broader quick rename -- change default endpoint name from 'http' to 'grpc' and change 'rest' to 'http' everywhere?

Reply:
I changed the default endpoint name to 'grpc' but left the other one 'rest`. I can rename it to 'http' everywhere if you'd like.

The part that I don't like about using 'http' is that they are both HTTP. Where at least "rest" gives a little more indication of the difference. But I can rename it to 'http' if you'd like.

Reply:
or the way you just did ;-)

Reply:
Looking for a little bit of previous art, Dapr says:

https://github.com/dotnet/aspire/blob/e2b1b46044bb74e9d7ccce54d00ada58d411b947/src/Aspire.Hosting.Dapr/DaprDistributedApplicationLifecycleHook.cs#L154-L156

Not saying we'd have to match it, but it at least shows we use this naming pattern elsewhere.

Reply:
I was thinking Qdrant docs being the winner. 😆 

I can honestly go either way. If I had to pick I would pick the way it is now in the PR mostly because it matches how Qdrant refers to them. But I can live with renaming "rest" => "http" if you'd like.

Reply:
Ugh, sorry not having this convo in original PR.  Let's rename to http -- the docs are mixed. But the **config** docs consistently use 'http_port' as what is being exposed, let's rename rest->http